cmake/ctest/drivers/atdm/cts1/drivers/Trilinos-atdm-cts1-intel-opt-openmp-panzer.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Should likely remove this file. We don't need a Panzer-only CTS-1 build at this point.